soh/src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -486,10 +482,13 @@ void func_80ABAC84(EnNiwLady* this, PlayState* play) {
     }
     osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ 正常終了 ☆☆☆☆☆ \n" VT_RST);
     if (LINK_IS_ADULT) {
-        if (!(gSaveContext.itemGetInf[2] & 0x1000)) {
-            gSaveContext.itemGetInf[2] |= 0x1000;
-        } else {
-            gSaveContext.itemGetInf[2] |= 0x4000;
+        // Flags for randomizer gives are set in the original message prompt choice handling
+        if (!gSaveContext.n64ddFlag) {
+            if (!(gSaveContext.itemGetInf[2] & 0x1000)) {
+                gSaveContext.itemGetInf[2] |= 0x1000;
```

Comment:
I must have missed these, could you convert them to Flags_SetItemGetInf calls?

Reply:
You didn't miss these. This is pointing to develop-sulu
Once we merge develop-sulu to develop, there will be expected merge conflicts with your flag changes.

Reply:
Aha gotcha